src/cai_lab_to_nwb/zaki_2024/interfaces/miniscope_imaging_interface.py

```diff
@@ -17,6 +18,101 @@
 from neuroconv.utils import DeepDict, dict_deep_update
 
 
+def get_session_start_time(folder_path: Union[str, Path]):
```

Comment:
Shouldn't we just pass the file path of the json directly instead of passing the folder and then locating it within? What do you think?

I know you are moving these functions only but is something to consider if you think that a small breaking change is worth avoiding the problem of a different file structure in the future.

Reply:
My first design choice was to pass the file path actually. I don't remember why I changed my mind, probably to avoid to repeat pice of code. But if you also think passing the file_path is the correct way, I will change it back.